aws-out-of-cluster.md

```diff
@@ -26,7 +26,7 @@ To allocate external AWS resources to a Kubernetes concept, use the following ta
 
 *\*In the `kubernetes_label_NAME` tag key, the `NAME` portion should appear exactly as the tag appears inside of Kubernetes. For example, for the tag `app.kubernetes.io/name`, this tag key would appear as `kubernetes_label_app.kubernetes.io/name`.*
 
-Additionally users must activate the `eks:cluster-name`
+Additionally users must activate the `eks:cluster-name` tag located in the AWS Managment Console under “AWS Billing > Cost allocation tags”.
```

Comment:
This line makes it sound lie this tag is required, but it's not clear to me why that's the case. Is this not an optional addition?

Reply:
It's also mentioned on the [Cloud Integrations (Line 33)](https://github.com/kubecost/docs/blob/af68e8a4c51657d2ed140fa60a1a6b9506dabc32/cloud-integration.md?plain=1#L33) page.  In that context it sound like it's only required when using the Beta `.Values.kubecostModel.cloudAssetsExcludeProviderID=true` value.

Still unclear to me if this is required for OOC costs, I'll continue to research.

Reply:
Good to update? Pretty sure this is optional!

Reply:
I think that might be required for the .Values.kubecostModel.cloudAssetsExcludeProviderID=true but some of that has been in flux. @Sean-Holcomb would know best.

Reply:
Sorry the wording is unclear there. It is required for Exclude Provider ID. It is needed for the "clever subtraction" aspect of the feature.

Reply:
I'm actually still not sure I know when exactly when it should be used. @Sean-Holcomb you ok to updating the wording or want to take @jcharcalla?